### DISCUSSION TITLE
balance bug in weaver profile

### DISCUSSION THREAD

**Discussion by mucar** [IBAX-io/go-ibax#21] [Bug Submission]
Hi, I changed my weaver profile name with a string (7 chars long). After this change my new balance is displayed as "Balance: 9999.998482 IBXC" instead of "Balance: 10000 IBXC". 

This operation should be free. Because there is a warning on profile name change page and it says: "Attention Letters, numbers, underscores, 1-20 characters. 1-3 characters, the cost is 10,000 tokens. 4 characters, the fee is 5000 tokens. 5 characters, the cost is 1000 tokens. 6~20 characters, free"

My wallet address is: 1168-2339-9243-7448-6536

**Comment by HalloKyle** [comment]
> Hi, I changed my weaver profile name with a string (7 chars long). After this change my new balance is displayed as "Balance: 9999.998482 IBXC" instead of "Balance: 10000 IBXC".
> 
> This operation should be free. Because there is a warning on profile name change page and it says: "Attention Letters, numbers, underscores, 1-20 characters. 1-3 characters, the cost is 10,000 tokens. 4 characters, the fee is 5000 tokens. 5 characters, the cost is 1000 tokens. 6~20 characters, free"
> 
> My wallet address is: 1168-2339-9243-7448-6536

Noted! Thanks for the report, please leave your Twitter handle and we can reward you with 500 IBAX points. Thank you, brother.

**Reply by HalloKyle** [reply]
> > Hi, I changed my weaver profile name with a string (7 chars long). After this change my new balance is displayed as "Balance: 9999.998482 IBXC" instead of "Balance: 10000 IBXC".
> > This operation should be free. Because there is a warning on profile name change page and it says: "Attention Letters, numbers, underscores, 1-20 characters. 1-3 characters, the cost is 10,000 tokens. 4 characters, the fee is 5000 tokens. 5 characters, the cost is 1000 tokens. 6~20 characters, free"
> > My wallet address is: 1168-2339-9243-7448-6536
> 
> Noted! Thanks for the report, please leave your Twitter handle and we can reward you with 500 IBAX points. Thank you, brother.

Free means if you name your address with 6-20 characters but calling contracts are always subject to gas deductions.

**Reply by mucar** [reply]
I see. But it is not mentioned in the paragraph that I quoted. This explanation can be added there. My twitter handle: @tfbugur. Thanks.
